bridge/server/services/data-service.ts

```diff
@@ -411,6 +411,20 @@ export class DataService {
     return tasks;
   }
 
+  public async getServiceNames(projectName: string): Promise<string[]> {
+    const resp = await this.apiService.getStages(projectName);
+    const stages = resp.data.stages;
+    const services: Map<string, string> = new Map();
```

Comment:
Because we don't need the value here, only the keys, I would use a Set. But if the insertion order is not used, I would prefer a plain object for unique values, because Map and Set are much slower.